cmd/traffic/cmd/manager/internal/cluster/info.go

```diff
@@ -242,6 +249,21 @@ func (oi *info) watchNodeSubnets(ctx context.Context, mustSucceed bool) bool {
 	return true
 }
 
+func getManagerSvcIP(ctx context.Context, env *managerutil.Env, client v1.CoreV1Interface) ([]byte, int32, error) {
+	sc, err := client.Services(env.ManagerNamespace).Get(ctx, env.AgentInjectorName, metav1.GetOptions{})
+	if err != nil {
+		return nil, 0, err
+	}
+	p := int32(0)
+	for _, port := range sc.Spec.Ports {
+		if port.Name == "https" {
+			p = port.Port
+			break
+		}
+	}
+	return net.ParseIP(sc.Spec.ClusterIP), p, nil
```

Comment:
Perhaps use `uputil.Parse` here instead. The `net.ParseIP` returns 16 byte IPs for IPv4.